### PR TITLE
Added prettier and eslint config files

### LINKED ISSUE #93: Chore: add eslint & prettier config
We are now using javascript for this project. It might be a good time to add some tools.
Should we keep the same code style as EddieBot?

### PR DESCRIPTION
**Pull request by Jashpatel1**
This issue relates to #93 

closes #93 

@AllanRegush @stemount @eddiejaoude

### CODE REVIEW

**Comment by AllanRegush** [comment]
Since we use svelte, I think we need some eslint & prettier plugins.

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "tabWidth": 2,
+    "useTabs": false,
+    "semi": true,
+    "singleQuote": true,
+    "printWidth": 80,
+    "plugins": ["prettier-plugin-svelte"]
+  }
```
Comment (BOLT04) [inline .prettierrc.json:8]:
```suggestion
  }
  
```

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "plugins": ["svelte3"],
+    "extends": ["eslint:recommended"],
+    "overrides": [
+        {
+        "files": ["**/*.svelte"],
+        "processor": "svelte3/svelte3"
+        }
+    ],
+    "rules": {
+    }
```
Comment (BOLT04) [inline .eslintrc.json:20]: we could add some rules that we'll have on EddieBot on this PR: https://github.com/EddieJaoudeCommunity/EddieBot/pull/287/files
```suggestion
    "rules": {
        "quotes": ["error", "single"],
        "indent": ["error", 2],
        "semi": ["error", "always"]
    }
```

**Review by BOLT04 — COMMENTED** [review]
LGTM 👍 just a few comments

**Review by BOLT04 — APPROVED** [review]
LGTM 👍, thank you @Jashpatel1 for working on this 😃.
Since you've added these config files, we can make a PR that formats everything to these rules later 👍

**Comment by eddiejaoude** [comment]
Great work everyone 👍

**Pull request merged**